external/js/cothority/lib/skipchain/skipchain.js

```diff
@@ -134,83 +136,107 @@ class Client {
    * @param {Object} BlockLink object (protobuf)
```

Comment:
Oh... ok. This will take a bit much time since that means I have to compute the message by hand, so I need to marshal a roster, which is not coded yet.

Reply:
maybe it's not important now, the roster in ForwardLink is often empty, we can do it later if it's necessary